Dockerfile

```
@@ -0,0 +1,10 @@
+FROM openjdk:17
+
+# JAR 파일의 경로를 ARG로 정의
+ARG JAR_FILE=build/libs/*.jar
+
+# JAR 파일을 컨테이너 내 app.jar로 복사
+COPY ${JAR_FILE} app.jar
```

Hadolint (2.12.0)

[warning] `COPY` to a relative destination without `WORKDIR` set.

(DL3045)